packages/console/src/components/PlanQuotaList/QuotaItem/index.tsx

```diff
@@ -17,23 +18,26 @@ type Props = {
   hasIcon?: boolean;
   quotaKey: keyof SubscriptionPlanQuota;
   quotaValue: SubscriptionPlanQuota[keyof SubscriptionPlanQuota];
+  isDiffItem?: boolean;
 };
 
-function QuotaDiffItem({ hasIcon = false, quotaKey, quotaValue }: Props) {
+function QuotaItem({ hasIcon, quotaKey, quotaValue, isDiffItem }: Props) {
   const { t } = useTranslation(undefined, { keyPrefix: 'admin_console.subscription.quota_item' });
   const isUnlimited = quotaValue === null;
   const isNotCapable = quotaValue === 0 || quotaValue === false;
   const isLimited = Boolean(quotaValue);
 
-  const Icon = isNotCapable ? Failed : DescendArrow;
+  const Icon = isNotCapable ? Failed : isDiffItem ? DescendArrow : Success;
```

Comment:
I think this needs to be refactored... It's become quite confusing now

Reply:
LOG-6571